src/org/anjocaido/groupmanager/GroupManager.java

```diff
@@ -151,14 +148,17 @@ public void onDisable(boolean restarting) {
 		if (!restarting) {
 			// Unregister this service if we are shutting down.
 			this.getServer().getServicesManager().unregister(this);
+			if (WorldEvents != null) WorldEvents = null;
+			BukkitPermissions = null;
+			GroupManager.logger.removeHandler(ch);
 		}
 
 		disableScheduler(); // Shutdown before we save, so it doesn't interfere.
 		if (worldsHolder != null) {
 			try {
 				worldsHolder.saveChanges(false);
 			} catch (IllegalStateException ex) {
-				GroupManager.logger.warning(ex.getMessage());
+				GroupManager.logger.log(Level.WARNING, ex.getMessage());

```

Comment:
This is functionally identical to the code it replaces. The only difference is this requires an import.